pinot-segment-local/src/main/java/org/apache/pinot/segment/local/utils/TableConfigUtils.java

```diff
@@ -1253,6 +1254,25 @@ public static void verifyHybridTableConfigs(String rawTableName, TableConfig off
           "Time column names are different for table: %s! Offline time column name: %s. Realtime time column name: %s",
           rawTableName, offlineTimeColumnName, realtimeTimeColumnName));
     }
+    TenantConfig offlineTenantConfig = offlineTableConfig.getTenantConfig();
+    TenantConfig realtimeTenantConfig = realtimeTableConfig.getTenantConfig();
+    Preconditions.checkNotNull(realtimeTenantConfig,
+        "Found null realtime table tenant config in hybrid table check for table: %s", rawTableName);
+    Preconditions.checkNotNull(offlineTableConfig,
+        "Found null offline table tenant config in hybrid table check for table: %s", rawTableName);
```

Comment:
(minor) These checks are redundant as they are already checked within the table config constructor

Reply:
i removed these test, thanks @Jackie-Jiang